src/actions/tags.js

```diff
@@ -112,10 +112,16 @@ export const clearSearchString = () => {
 * Action to reset all tags to empty
 */
 
-export const resetTags = (displayName, id) => {
```

Comment:
But the resetTags function is called by the socket service and by the tag component. This means that the default tag is declared in two places. The default tag should only need to be specified in one place so it has been moved to the reducer and removed as an argument.